sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -5586,6 +5621,23 @@ export interface ContainerChangeLeaseOptions extends CommonOptions {
   conditions?: ModifiedAccessConditions;
 }
 
+/**
+ * Options to configure the {@link ContainerClient.deleteBlob} operation.
+ *
+ * @export
+ * @interface ContainerDeleteBlobOptions
+ */
+export interface ContainerDeleteBlobOptions extends BlobDeleteOptions {
```

Comment:
This is a good extension, will this be used across language? Looks it's addition to existing discussion?

Reply:
They will review this PR. So will discuss here if needed.

Reply:
> > deleteSnapshots?: DeleteSnapshotsOptionType;
> 
> Please check what will happen if deleteSnapshots used with versionId.
> 
> Refers to: sdk/storage/storage-blob/src/Clients.ts:356 in [5232034](https://github.com/Azure/azure-sdk-for-js/commit/5232034e9e7aae5e3c319d722ad71ee747a6f17d). [](commit_id = [5232034](https://github.com/Azure/azure-sdk-for-js/commit/5232034e9e7aae5e3c319d722ad71ee747a6f17d), deletion_comment = False)

Done. And both failed with "This operation is only allowed on the root 
blob. Version id should not be provided".